environment.yml

```diff
@@ -1,104 +1,162 @@
 name: intake-xarray
```

Comment:
This is hard to see - only the indentation changed, or something more?

Reply:
Yes, this had gone stale and I was just updating the dependencies.